src/test/rgw/bucket_notification/README.rst

```diff
@@ -0,0 +1,43 @@
+============================
+ Bucket Notification tests
+============================
+
+The tests use the Nose test framework. To get started, ensure you have
+the ``virtualenv`` software installed; e.g. on Debian/Ubuntu::
+
+        sudo apt-get install python-virtualenv
+
+and then run::
+
+        ./bootstrap
+
+You will need to use the sample configuration file named ``bntests.conf.SAMPLE`` 
+that has been provided at ceph/src/test/rgw/bucket_notification. This file can be
+used to run the bucket notification tests on a Ceph cluster started with vstart.
+
+You also need to install Kafka which can be done by downloading and unzipping from the following::
+
+        https://www.apache.org/dyn/closer.cgi?path=/kafka/2.6.0/kafka-2.6.0-src.tgz
```

Comment:
link is broken. please use: `https://archive.apache.org/dist/kafka/2.6.0/kafka-2.6.0-src.tgz`